tests/tests/test_manage_pgpool2.py

```diff
@@ -0,0 +1,118 @@
+import pytest
+
+from conftest import (
+    load_ansible_vars,
+    get_pg_type,
+    get_pgpool2,
+    get_pg_version,
+    get_primary
+)
+
+def test_manage_pgpool_pcp_user():
+    #PCP commands only available for pg-pool-4.3 which is only available for EPAS
+    if get_pg_type() == 'PG':
+        pytest.skip()
```

Comment:
That's interesting. Actually, we must install the package `pgpool-II-pcp`. Can you fix that? This package must be installed by the role `setup_pgpool2`.